sauce/features/accounts/toggle-transaction-filters/index.js

```diff
@@ -1,27 +1,31 @@
 import { Feature } from 'toolkit/core/feature';
-import * as toolkitHelper from 'toolkit/helpers/toolkit';
+// import * as toolkitHelper from 'toolkit/helpers/toolkit';
+import { getCurrentRouteName } from 'toolkit/helpers/toolkit';
+import { controllerLookup } from 'toolkit/helpers/toolkit';
```

Comment:
can remove the comment and change this to be

`import { controllerLookup, getCurrentRouteName } from 'toolkit/helpers/toolkit';`


:fancy: